content/departments/cloud/index.md

```diff
@@ -1,12 +1,12 @@
-# Cloud
+# Cloud Opreations Team
 
 > NOTE: **Cloud means single-tenant dedicated instances managed by Sourcegraph** _(for example `mycompany.sourcegraph.com`)_. Sourcegraph Cloud should not be confused with Sourcegraph.com which holds public and open source code. The Cloud and managed instance should be considered synonyms within these handbook pages.
 
 The Cloud team is the special focus team reporting directly to CEO modeled on _“if AWS were to offer ‘Managed Sourcegraph’ like they do Elasticsearch, Redis, PostgreSQL, etc., how would they do it?”_ The team is responsible for maintaining existing [managed instances](https://docs.sourcegraph.com/admin/install/managed) and building the next generation of them. The Cloud team has no other responsibilities.
```

Comment:
some feedback to the Cloud section:

I think in the long run, we want Cloud to become the platform team and offload On-Call responsibility back to the home team. Is this something we want to capture here?

Reply:
We can. This is the team area, so as long as the team and leadership are aligned, it sounds like a good idea.